ci-operator/step-registry/hypershift/kubevirt/install/hypershift-kubevirt-install-ref.yaml

```diff
@@ -7,6 +7,9 @@ ref:
   - name: CNV_PRERELEASE_VERSION
     default: ""
     documentation: "version of the kubevirt. Empty by default it will use stable"
+  - name: CNV_PRERELEASE_CATALOG_IMAGE
+    default: ""
```

Comment:
I think maybe it woild be more clear to set the default image here instead of in the script body. e.g. the default value is published in the step documentation. WDYT?

Reply:
that would require us to create a default for the CNV_PRERELEASE_VERSION as well, which we don't want to do because leaving CNV_PRERELEASE_VERSION empty results in the latest stable cnv being used.